frame/composable-traits/src/clearing_house.rs

```diff
@@ -77,10 +77,32 @@ pub trait ClearingHouse {
 		base_asset_amount_limit: Self::Balance,
 	) -> Result<Self::Balance, DispatchError>;
 
-	/// Update the funding rate for a market
+	/// Close an existing position in a market.
+	///
+	/// This is an alternative to calling [`open_position`](Self::open_position) with an opposite
+	/// direction to the existing position, a `quote_asset_amount` equivalent to the absolute value
```

Comment:
I think there is an "of: missing after "absolute value"

Reply:
that's correct, good catch